docs/release_guide.md

```diff
@@ -52,7 +52,6 @@ Create a new commit on the new branch to update the hardcoded version informatio
 
 * The container image tag in [charts/osm/values.yaml](https://github.com/openservicemesh/osm/tree/release-v0.8/charts/osm/values.yaml)
 * The chart and app version in [charts/osm/Chart.yaml](https://github.com/openservicemesh/osm/tree/release-v0.8/charts/osm/Chart.yaml)
-* The default osm-controller image tag in [osm cli install](https://github.com/openservicemesh/osm/blob/release-v0.8/cmd/cli/install.go)
```

Comment:
nit: deletion of this should not be a part of this pr

Reply:
Reverted